libraries/AP_ADSB/AP_ADSB_Sagetech_MXS.h

```diff
@@ -0,0 +1,268 @@
+/*
+    Copyright 2022 Sagetech Avionics Inc. All rights reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    https://github.com/Sagetech-Avionics/sagetech-mxs-sdk/blob/main/doc/pdf/ICD02373_MXS_Host_ICD.pdf
+
+    Author: Chuck Faber, Tom Pittenger
+*/
+
+#pragma once
+
+#include "AP_ADSB_Backend.h"
+
+#ifndef HAL_ADSB_SAGETECH_MXS_ENABLED
+#define HAL_ADSB_SAGETECH_MXS_ENABLED HAL_ADSB_ENABLED
+#endif
+
+#if HAL_ADSB_SAGETECH_MXS_ENABLED
+
+#include "sagetech-sdk/sagetech_mxs.h"

```

Comment:
we will need to carefull check this one as it inclue sgUtils.h that got a lot of define that we are already defining ourself, we don't want them to propagate outside this ADSB backend !

Reply:
I can wrap these defines in #ifndefs if that helps.